app/src/main/java/com/zulip/android/activities/ZulipActivity.java

```diff
@@ -1633,6 +1637,28 @@ public void clearSearch() {
         inSearch = false;
     }
 
+    /**
+     * dismiss toast
+     */
+    private void dismissToast() {
+        try {
+            toast.cancel();
+        } catch (NullPointerException e) {
```

Comment:
hear i think we could handle this error gracefully by `if (toast != null)` check and not send the traceback to crashylytics using `Zlog.logException(e)` :)